src/components/ProjectSettingsPage/index.jsx

```diff
@@ -79,7 +79,7 @@ class ProjectSettingsPage extends React.Component {
   }
 
   handleSubmit() {
-    const { projectName, projectDescription } = this.state;
+    var { projectName, projectDescription } = this.state;
```

Comment:
@khalifan-kfan we try to make our code more error-proof everywhere as const is an explicit constant reference, which means it has a strict type, so you won't accidentally reassign variables. Var allows the developer to make unintentional mistakes, whereas let and const are designed to make devs consider the variables' long-time mutability, leading to fewer accidental errors, and given these are variables from state, the mutability is only handled by the "setState" method. Do you have a reason why we need to use var?

Reply:
Oh thanks let me handle it differently

Reply:
@neelxie 
tried to trim in onChangeText()'s setstate  but it could not let the user  edit  freely so
the other approach is a bit repetitive  but does the job, is it alright ?

Reply:
@neelxie The new commit has a better approach,

Reply:
@neelxie The new commit has a better approach,